Buildings/Templates/ChilledWaterPlant/Components/CoolingTowerGroup/package.mo

```diff
@@ -0,0 +1,5 @@
+within Buildings.Templates.ChilledWaterPlant.Components;
+package CoolingTowerGroup
```

Comment:
I'm going to hold off on the renaming. I agree with your suggestion, but I think all of the components could be renamed (and pluralized), and this might mess up the review since github might not recognize that the files are renamed and assume that files are deleted and new files are created.